app/os.go

```diff
@@ -45,6 +45,12 @@ type Config struct {
 	CustomRenderer bool
 	// Decorated reports whether window decorations are provided automatically.
 	Decorated bool
+
+	// HiddenMinimizeButton hide the window's minimize button
+	HiddenMinimizeButton bool
+	// HiddenMaximizeButton hide the window's maximize button
+	HiddenMaximizeButton bool
```

Comment:
I think you should either note that these don't work under Linux, or make them work under Linux.

Reply:
I have tried for days to test on ubuntu with os_x11.go, but still not working.
- _NET_WM_ALLOWED_ACTIONS
- _MOTIF_WM_HINTS
- _NET_WM_WINDOW_TYPE: _NET_WM_WINDOW_TYPE_DIALOG

They all didn't work. I belive these properties also depend on window manager implementation, according to infomations bellow:
- https://specifications.freedesktop.org/wm-spec/1.3/ar01s05.html
- https://stackoverflow.com/questions/14442081/disable-actions-move-resize-minimize-etc-using-python-xlib
- https://superuser.com/questions/186748/how-to-hide-or-minimize-x11-window-from-console

I'll continue wokring to make them work on x11 and wayland, but this may take long time.
So I add tips "only works for windows and macOS" for the options.
Maybe the feature working only on some os is acceptable?

Reply:
> Maybe the feature working only on some os is acceptable?

Fine with me. Better than nothing! Thanks for trying.